src/services/dissolution/dissolution.service.ts

```diff
@@ -35,4 +37,13 @@ export default class DissolutionService {
 
     return await this.client.getDissolution(token, companyNumber)
   }
+
+  public async approveDissolution(token: string, dissolutionSession: DissolutionSession, email: string): Promise<DissolutionPatchResponse> {
```

Comment:
This should return `Promise<void>` as we don't need anything from the `DissolutionPatchResponse`. The service should only return the fields we care about, and in the frontend we care about none of them.

The only reason that the client returns the `DissolutionPatchResponse` is because it is a HTTP wrapper.